src/haddock/modules/analysis/caprieval/defaults.yaml

```diff
@@ -1,157 +1,186 @@
+    If none is defined then the lowest scoring structure is selected by default.
+  group: 'analysis'
   explevel: easy
 irmsd:
   default: true
   type: boolean
-  title: No title yet
-  short: No short description yet
-  long: No long description yet
-  group: ''
+  title: Calculate the I-RMSD
+  short: Perform I-RMSD calculations.
+  long: Perform I-RMSD calculations.
+  group: 'analysis'
   explevel: easy
 fnat:
   default: true
   type: boolean
   title: No title yet
-  short: No short description yet
-  long: No long description yet
-  group: ''
+  short: Perform FNAT calculations.
+  long: Perform FNAT calculations.
+  group: 'analysis'
   explevel: easy
 lrmsd:
   default: true
   type: boolean
   title: No title yet
-  short: No short description yet
-  long: No long description yet
-  group: ''
+  short: Perform L-RMSD calculations.
+  long: Perform L-RMSD calculations.
+  group: 'analysis'
   explevel: easy
 ilrmsd:
   default: true
   type: boolean
   title: No title yet
-  short: No short description yet
-  long: No long description yet
-  group: ''
+  short: Perform I-L-RMSD calculations.
+  long: Perform I-L-RMSD calculations.
+  group: 'analysis'
   explevel: easy
 dockq:
   default: true
   type: boolean
   title: No title yet
-  short: No short description yet
-  long: No long description yet
-  group: ''
+  short: Evaluate DockQ.
+  long: Evaluate DockQ.
+  group: 'analysis'
   explevel: easy
 irmsd_cutoff:
   default: 10.0
   type: float
-  min: -9999
+  min: 0.0
   max: 9999
   precision: 3
-  title: No title yet
-  short: No short description yet
-  long: No long description yet
-  group: ''
+  title: Distance cutoff (Å) to consider the interface
+  short: Distance cutoff (Å) to consider the interface.
+  long: Distance cutoff (Å) to consider the interface.
+  group: 'analysis'
   explevel: easy
 fnat_cutoff:
   default: 5.0
   type: float
-  min: -9999
+  min: 0.0
   max: 9999
   precision: 3
-  title: No title yet
-  short: No short description yet
-  long: No long description yet
-  group: ''
+  title: Distance cutoff (Å) to consider the interface
+  short: Distance cutoff (Å) to consider the interface.
+  long: Distance cutoff (Å) to consider the interface.
+  group: 'analysis'
   explevel: easy
 lrmsd_cutoff:
   default: 10.0
   type: float
-  min: -9999
+  min: 0.0
   max: 9999
   precision: 3
-  title: No title yet
-  short: No short description yet
-  long: No long description yet
-  group: ''
+  title: Distance cutoff (Å) to consider the interface
+  short: Distance cutoff (Å) to consider the interface.
+  long: Distance cutoff (Å) to consider the interface.
+  group: 'analysis'
   explevel: easy
 receptor_chain:
   default: A
   type: string
-  minchars: 0
-  maxchars: 100
-  title: No title yet
-  short: No short description yet
-  long: No long description yet
-  group: ''
+  minchars: 1
+  maxchars: 1
+  title: Receptor ChainID
+  short: Receptor ChainID.
+  long: Receptor ChainID.
```

Comment:
Relevant for the l-RMSD and i-l-RMSD calculations. So this should be explained. After all this is a "long" explanation...

Reply:
updated